packages/webapp/public/locales/fr/translation.json

```diff
@@ -497,10 +497,13 @@
   "EXPENSE": {
     "ADD_EXPENSE": {
       "ALL_FIELDS_REQUIRED": "Tous les champs sont requis",
+      "FLOW": "Création d'une dépense",
       "MIN_ERROR": "Veuillez saisir une valeur supérieure à ",
       "REQUIRED_ERROR": "Une dépense est requise",
+      "TITLE": "Ajouter une dépense",
       "TITLE_1": "Nouvelle dépense (1 sur 2)",
-      "TITLE_2": "Nouvelle dépense (2 sur 2)"
+      "TITLE_2": "Nouvelle dépense (2 sur 2)",
+      "WHICH_TYPES_TO_RECORD": "Quel type de dépense voulez-vous enregistrer ?"
```

Comment:
I think this needs the non-breaking space code \u00a0 before the ? as used elsewhere in the fr translation file or the ? can end up floating alone on mobile, which I think is meant to be avoided:

<img width="396" alt="Screenshot 2023-08-31 at 9 05 55 AM" src="https://github.com/LiteFarmOrg/LiteFarm/assets/82857964/eb5604a6-a834-485f-8e30-2209cc718d4d">

Reply:
I wasn't aware of this, thank you!